ui/raidboss/data/03-hw/trial/sophia-ex.txt

```diff
@@ -1,157 +1,147 @@
+12.0 "Thunder II" sync /:Sophia:19B0:/ window 12,5
+22.4 "Gnosis" sync /:Barbelo:19C2:/
+41.4 "Arms Of Wisdom?" sync /:Sophia:19C4:/ # May be skipped depending on DPS
+
+# This section will be reached either at 56.0, or at the time she is pushed to 90%
+56.0 "--Clones Appear--" sync / 03:........:Added new combatant Aion Teleos/ window 56,5
+63.6 "Thunder II/Thunder III" sync /:Sophia:19(AC|B0):/
+63.6 "Duplicate" sync /:Aion Teleos:19AB:/ window 30,2.5
+69.7 "Aero III" sync /:Sophia:19AE:/
+69.7 "Duplicate" sync /:Aion Teleos:19AB:/
+79.8 "Execute" sync /:Sophia:19AA:/
+#79.8 "Execute" sync /:Aion Teleos:19AF:/
+90.2 "Arms Of Wisdom" sync /:Sophia:19C4:/
+
+# Intermission.
+# Divine Spark can be pushed and thus is not synced.
+# Gnostic Rant and Infusion are purely HP-based pushes.
+110.6 "Cloudy Heavens" sync /:Sophia:19BE:/ window 110.6
+111.6 "--Untargetable--"
+120.9 "Horizontal Kenoma/Vertical Kenoma" sync /:The First Demiurge:19B[BC]:/
+121.9 "Gnostic Spear" sync /:The Third Demiurge:19B9:/
+125.9 "Divine Spark" # sync /:The Second Demiurge:19B6:/
+129.0 "Ring Of Pain" sync /:The Third Demiurge:19BA:/
+135.1 "Horizontal Kenoma/Vertical Kenoma" sync /:The First Demiurge:19B[BC]:/
+147.3 "Gnostic Spear?" sync /:The Third Demiurge:19B9:/
+149.2 "Horizontal Kenoma/Vertical Kenoma" sync /:The First Demiurge:19B[BC]:/
+191.6 "Zombification Enrage?"
+
+# There's an introductory block leading into a rotation.
+250.0 "The Scales Of Wisdom" sync /:Sophia:1981:/ window 250,5
+267.5 "The Scales Of Wisdom" sync /:Sophia:1AE1:/
+272.9 "--Targetable--"
+280.8 "Quasar (Meteor Snapshot)" sync /:Sophia:19(6E|A7):/
+286.8 "Quasar (Meteor Detonate)" sync /:Sophia:1A87:/
+291.3 "Aero III/Thunder II/Thunder III" sync /:Sophia:19(A[CE]|BC):/ window 41.3,5
+297.1 "Quasar Tethers" sync /:Sophia:196E:/
+305.7 "Quasar (Tilt)" sync /:Sophia:1A4C:/
+310.7 "Cintamani x2" duration 3 # sync /:Sophia:19C5:/
+318.8 "Arms Of Wisdom" sync /:Sophia:19C4:/
+ 
+ # Potential landing point if DPS is high. Push seems to be 70%?
+328.3 "Thunder II" sync /:Sophia:19B0:/ window 68.8,5
+334.0 "Quasar Tethers" sync /:Sophia:19A7:/
+335.0 "--Untargetable--"
+341.2 "Onrush" sync /:Sophia:19C1:/
+341.8 "Quasar (Tilt)" # sync /:Sophia:1A4C:/
+345.4 "--Targetable--"
+361.5 "Dischordant Cleansing" # sync /:Sophia:19B[35]:/
+361.5 "Cintamani x2" duration 3 # sync /:Sophia:19C5:/
+370.8 "Arms Of Wisdom" sync /:Sophia:19C4:/ window 30,5
+379.0 "--Clones Appear--" sync / 03:........:Added new combatant Aion Teleos/ window 60,20
+382.0 "Cintamani x3" duration 6 # sync /:Sophia:19C5:/
+393.3 "Duplicate" sync /:Aion Teleos:19AB:/
+393.3 "Thunder II/Thunder III" sync /:Sophia:19(AC|B0):/
+399.4 "Duplicate" sync /:Aion Teleos:19AB:/
+399.4 "Aero III" sync /:Sophia:19AE:/ window 30,5
+408.5 "Arms Of Wisdom" sync /:Sophia:19C4:/
+419.8 "Light Dew" sync /:Barbelo:19BF:/
+420.5 "Execute" sync /:Sophia:19AA:/ window 10,10
+
+# Rotation block.
+# Technically it can loop, but unless party DPS is VERY high,
```

Comment:
Is it worth just making it loop to avoid writing it out a second time?

Reply:
We can do that, sure. I did this because it made the jump for the loop clean, but we could do a fake lookahead and sync from there. I typically avoid doing that but we can here.